tests/mythril/mythril_leveldb_test.py

```diff
@@ -0,0 +1,45 @@
+import io
+import pytest
+from contextlib import redirect_stdout
+from pathlib import Path
+
+from mythril.mythril import MythrilLevelDB, MythrilConfig
+from mythril.exceptions import CriticalError
+
+config = MythrilConfig()
+config.set_api_leveldb(
+    str(Path(__file__).parent.parent / "testdata/leveldb_dir/geth/chaindata")
+)
+
+
+def test_leveldb_code_search():
+    leveldb_search = MythrilLevelDB(leveldb=config.eth_db)
```

Comment:
Why not use a mock of a leveldb object? The benefits are that we test MythrilLevelDB in isolation & we probably don't get those deprecation warnings for leveldb stuff

Reply:
The entire mythril-leveldb lib didn't have any tests. So i thought one direct run of the library might be a good idea as I found it wasn't working from mythril `v0.18.13`  due to some bug.

Reply:
Do you mean the mythril side or the leveldb python package?

Reply:
mythril side